ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource

```diff
@@ -386,13 +386,8 @@ Deploy Kserve Model Via UI  #robocop: disable
     [Documentation]  Deploys a model using the kserve/caikit runtime using the Data Science Projects UI
     [Arguments]    ${model_name}    ${serving_runtime}    ${data_connection}    ${path}
     ...    ${model_framework}=caikit    ${replicas}=1    ${size}=Small    ${no_gpus}=${0}
-    ${first_deploy}=    Run Keyword And Return Status
-    ...    Page Should Contain Element    xpath=//article[@id="single-serving-platform-card"]
-    IF  ${first_deploy}
-        Click Button    //article[@id="single-serving-platform-card"]//button[.="Deploy model"]
-    ELSE
-        Click Button    //div[@id="model-server"]//button[.="Deploy model"]
-    END
+    Move To Tab    Models
+    Click Button    //button[@data-testid="single-serving-deploy-button"]
```

Comment:
Just to be sure - does this work also in case this isn't the first model deploy?

Reply:
Previously for "first deploy" it clicked on `//article[@id="single-serving-platform-card"]//button[.="Deploy model"]`and otherwise on `//div[@id="model-server"]//button[.="Deploy model"]`.
Now the "Deploy model" is available via //button[@data-testid="single-serving-deploy-button"], so I simply click on it.

Reply:
Updated and retested